### PR TITLE
fix: @adobe/helix-fetch -> @adobe/fetch

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by stefan-guggisberg** [comment]
@trieloff The tests are failing randomly. I don't know what's wrong.

**Comment by tripodsan** [comment]
> @trieloff The tests are failing randomly. I don't know what's wrong.

I suggest that streamline this, and remove all edge compute and gateway integration.

**Comment by trieloff** [comment]
> I suggest that we streamline this, and remove all edge compute and gateway instrumentation.

I'm using edge compute for `helix-rum-collector` and Gateway for `helix-run-query`

File: src/bundler/EdgeBundler.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ export default class EdgeBundler extends WebpackBundler {
           // the main.js is imported in the universal adapter and is _the_ action entry point
           './main.js': cfg.file,
           // 'psl': path.resolve(__dirname, '../node_modules/psl/dist/psl.js'), // inlined data
-          '@adobe/helix-fetch': path.resolve(__dirname, '../template/polyfills/helix-fetch.js'),
+          '@adobe/fetch': path.resolve(__dirname, '../template/polyfills/fetch.js'),
```
Comment (trieloff) [inline src/bundler/EdgeBundler.js:68]:
```suggestion
          '@adobe/fetch': path.resolve(__dirname, '../template/polyfills/fetch.js'),
          '@adobe/helix-fetch': path.resolve(__dirname, '../template/polyfills/fetch.js'),
```

we'd want both libraries to be poly filled with the runtime-provided fetch implementation.

**Review by trieloff — APPROVED** [review]

**Review by tripodsan — APPROVED** [review]

**Pull request merged**